apps/framework-cli/src/project.rs

```diff
@@ -170,9 +154,8 @@ impl Project {
         }
     }
 
-    pub fn set_enviroment(&self, production: bool) {
-        let mut proj = PROJECT.lock().unwrap();
-        proj.is_production = production;
+    pub fn set_enviroment(&mut self, production: bool) {
```

Comment:
Not sure about this naming.  set_environment doesn't seem to be specific enough - set_production_env or set_prod_env would result in the boolean argument value making sense while reading code.

Reply:
Good call! will change